lessons/beginners/list/index.md

```diff
@@ -354,25 +389,34 @@ for cislo in range(10):
 print(mocniny_dvou)
 ```
 
+Podobným způsobem získáš seznam seznam `matka`, `babička`, `prababička`,
+`praprababička`, atd.:
+
+```python
+predkove = ['matka']
+for pocet_pra in range(10):
+    predkove.append('pra' * pocet_pra + 'babička')
+print(predkove)
+```
+
 Chceš-li seznam, který reprezentuje balíček karet,
-zavolej `append` pro všechny kombinace barev a hodnot.
+zavolej `append` pro všechny kombinace barev a hodnot:
 
 ```python
 balicek = []
-for barva in '♠', '♥', '♦', '♣':  # (Na Windows použij textová jména)
+for barva in '♠', '♥', '♦', '♣':
     for hodnota in list(range(2, 11)) + ['J', 'Q', 'K', 'A']:
```

Comment:
_itertools.product_ asi ne, co?

Jinak ten výraz jako celek mi přijde docela kryptický.

```suggestion
ciselne_hodnoty = list(range(2, 11))
pismenne_hodnoty = ['J', 'Q', 'K', 'A']
for barva in '♠', '♥', '♦', '♣':
    for hodnota in ciselne_hodnoty + pismenne_hodnoty:
```

Reply:
A nebo to rozepsat úplně?
```suggestion
for barva in '♠', '♥', '♦', '♣':
    for hodnota in ['2', '3', '4', '5', '6', '7', '8', '9', '10', 'J', 'Q', 'K', 'A']:
```

Reply:
... a zjednodušení pak dát za úkol?

Reply:
Pro někoho, kdo před chvilkou poprvé uviděl převádění na seznamy, je ten kód opravdu složitý. Zjednodušil jsem ho co do konceptů, tedy s tím `['2', '3', '4', '5', '6', '7', '8', '9', '10', 'J', 'Q', 'K', 'A']`, a pak se v poznámce rozepsal o tom, jak to udělat „správně“.

Reply:
Jako vlastně proč ne. Je sice pravda, že se jedná o číselnou řadu, ale na druhou stranu je těch hodnot jen pár, jsou to karty, nezačínají jedničkou… Může bejt.